sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/Combine.java

```diff
@@ -254,10 +254,26 @@ private Combine() {
    *
    * <pre>{@code
    * public class AverageFn extends CombineFn<Integer, AverageFn.Accum, Double> {
-   *   public static class Accum {
+   *   public static class Accum implements java.io.Serializable {
```

Comment:
If you want to fully qualify Serializable, you'll want to fully qualify all the other types such as CombineFn, Combine, and PCollection which that are part of the example. Consider dropping the `java.io.` part instead.

Reply:
Ack.